connector/protobuf/pom.xml

```diff
@@ -92,7 +92,7 @@
         <artifactId>maven-shade-plugin</artifactId>
         <configuration>
           <shadedArtifactAttached>false</shadedArtifactAttached>
-          <shadeTestJar>true</shadeTestJar>
+          <shadeTestJar>false</shadeTestJar>
```

Comment:
[22c0173](https://github.com/apache/spark/pull/40738/commits/22c0173793ae5b978103bb2050e66bd72336d86f) hang when maven shade to downloads some non-existent tests. jar and set it to false to try again